app/build.gradle

```diff
@@ -13,13 +13,21 @@ android {
             jsonFile = file('app-publisher.json')
         }
     }
+    signingConfigs {
+        release {
+            storeFile file('../SunsetScrob.jks')
+            storePassword 'o}6hp(payMq'
+            keyAlias "sunsetScrob"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/225c55a0de3a831c6402a996175e779bd24a6619/app/build.gradle#L20"></span>Not targeting the latest versions of Android; compatibility <br />
modes apply. Consider testing and updating this version. <br />
Consult the android.os.Build.VERSION_CODES javadoc for details.</td>
    </tr>
  </tbody>
</table>